bin/cli.js

```diff
@@ -1,3 +1,30 @@
 #!/usr/bin/env node
 
-require('../build/src/cli.js')
+var devMode = require('fs').existsSync(`${__dirname}/../src`);
```

Comment:
This assumes that the src folder at the root isn't included in the built package.

I have in the past allowed a developer to specify they still want to use the compiled scripts when running locally, but it doesn't seem very useful.

Reply:
this is nice, but the npm ignore should then ignore the src folder